modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -379,7 +395,42 @@ sda.enkf.multisite <- function(settings,
       if (t>1){
         #for next time step split the met if model requires
         #-Splitting the input for the models that they don't care about the start and end time of simulations and they run as long as their met file.
-        inputs.split <- metSplit(conf.settings, inputs, settings, model, no_split = FALSE, obs.times, t, nens, restart_flag = FALSE, my.split_inputs)
+        #set start and end date for splitting met
```

Comment:
Conceptually, I'm not seeing why you're replacing a simple call to a function with a big chunk of code, which undo's the concept of modularization and refactoring. If there's something wrong within the function, I'd fix the function.

Also, conceptually note that MOST models don't need their met split, so make sure that any implementation of metSplit

Reply:
Previously, I directly modified "metSplit" but it did not seem to be suggested. The challenge is it that we only need to do the split function for "met" sublist but now it lacks the specification: https://github.com/PecanProject/pecan/blob/develop/modules/assim.sequential/R/metSplit.R#L39. Do you suggest we change it to:"           inputs.split$met$samples[i] <- do.call(
            my.split_inputs,
            args = list(
              settings = settings,
              start.time = (lubridate::ymd_hms(start.time, truncated = 3) + lubridate::second(lubridate::hms("00:00:01"))),
              stop.time =   lubridate::ymd_hms(stop.time, truncated = 3),
              inputs = inputs$met$samples[[i]])